src/cace/amm/obj_ns.c

```diff
@@ -143,3 +143,15 @@ cace_amm_obj_desc_t *cace_amm_obj_ns_find_obj_enum(const cace_amm_obj_ns_t *ns,
     }
     return *found;
 }
+
+bool cace_amm_obj_ns_is_odm(const cace_amm_obj_ns_t *ns)
+{
+    CHKFALSE(ns);
+
+    if (ns->model_id.has_intenum && ns->model_id.intenum < 0) // FIXME: check if name starts with !
```

Comment:
There is an existing boolean function `m_string_start_with_str_p(name, "!")` to check the name

Reply:
Thanks, updated to include this check.